src/client/app/containers/app.ts

```diff
@@ -73,12 +73,13 @@ export class AppComponent {
         return this.profile$
             .filter(profile => profile != null)
             .map(profile => {
-                if (!isNil(this.snippet.gistOwnerId)) {
-                    return this.snippet.gistOwnerId === profile.login;
+                if (isNil(this.snippet.gist)) {
```

Comment:
Then `isEmpty` is a super set of `isNil`. I'd use that instead as if the gist is empty, you don't want to do anything anyways.